src/components/OSCALControl.js

```diff
@@ -70,41 +70,40 @@ export default function OSCALControl(props) {
             </Typography>
           </Grid>
         </Grid>
-        {props.control.parts &&
-          props.control.parts.map((part, index) => (
-            <OSCALControlPart
-              part={part}
-              control={props.control}
-              controlId={props.control.id}
-              parameters={props.control.params}
-              implementedRequirement={props.implementedRequirement}
-              componentId={props.componentId}
-              modificationAlters={props.modificationAlters}
-              modificationSetParameters={props.modificationSetParameters}
-              key={part.id ?? `part-${index}`}
-              isEditable={props.isEditable}
-              onRestSuccess={props.onRestSuccess}
-              onRestError={props.onRestError}
-              partialRestData={props.partialRestData}
-            />
-          ))}
-        {props.control.controls &&
-          props.control.controls.map((control) => (
-            <OSCALControl
-              control={control}
-              parameters={control.params}
-              includeControlIds={props.includeControlIds}
-              modificationAlters={props.modificationAlters}
-              modificationSetParameters={props.modificationSetParameters}
-              childLevel={(props?.childLevel ?? 0) + 1}
-              key={control.id}
-              implementedRequirement={props.implementedRequirement}
-              isEditable={props.isEditable}
-              onRestSuccess={props.onRestSuccess}
-              onRestError={props.onRestError}
-              partialRestData={props.partialRestData}
-            />
-          ))}
+        {props.control.parts?.map((part, index) => (
+          <OSCALControlPart
+            part={part}
+            control={props.control}
+            controlId={props.control.id}
+            parameters={props.control.params}
+            implementedRequirement={props.implementedRequirement}
+            componentId={props.componentId}
+            modificationAlters={props.modificationAlters}
+            modificationSetParameters={props.modificationSetParameters}
+            key={part.id ?? `part-${index}`}
+            isEditable={props.isEditable}
+            onRestSuccess={props.onRestSuccess}
+            onRestError={props.onRestError}
+            partialRestData={props.partialRestData}
+          />
+        ))}
+        {props.control.controls?.map((control) => (
+          <OSCALControl
+            control={control}
+            parameters={control.params}
+            includeControlIds={props.includeControlIds}
+            modificationAlters={props.modificationAlters}
+            componentId={props.componentId}
```

Comment:
This is the major change. I went ahead and cleaned up the `map` a bit as well.

Reply:
Since you're making a mess of the diff anyway :P can you update these to pass the props in a more similar/identical order?

Reply:
It is now in alphabetical order!